### PR TITLE
Full Travis CI configuration

### PR DESCRIPTION
**Pull request by jeffposnick**
R: Any/all

This should be enough to get a Travis CI build going for the Jekyll content that's checked in to any branch.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,2 +1,10 @@
 language: ruby
+rvm:
+- 2.1
 script: "bundle exec jekyll build"
```
Comment (PaulKinlan) [inline .travis.yml]: I would change this to 
script:
- bundle exec jekyll build

it just helps for the future.

Comment (PaulKinlan) [inline .travis.yml]: Also are we absolutely sure that jekyll is deployed on the travis infrastructure?  It seems to make sense to have an

install:
- bundle install

Comment (jeffposnick) [inline .travis.yml]: Travis CI automatically runs that for you when it detects that there's a `Gemfile`: http://docs.travis-ci.com/user/languages/ruby/#Travis-CI-uses-Bundler

You can't use `cache: bundler` if you override the install step: http://docs.travis-ci.com/user/caching/#With-a-custom-install-step, so it's cleaner to leave things as-is and take advantage of the default behavior.

**Comment by jeffposnick** [comment]
I'm going to merge, having made the change @PaulKinlan suggested.

**Pull request merged**